application/controllers/collection_instrument.py

```diff
@@ -453,7 +460,17 @@ def get_instrument_data(instrument_id, session):
 
         data = None
         file_name = None
+
         if instrument:
+            if instrument.file_location is not None:
```

Comment:
I think you could just do `if instrument.file_location` as None is falsy in pyton